docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML authentication in UI
+title: Configure SAML authentication in Grafana's UI
+weight: 1150
+---
+
+# Configure SAML authentication in Grafana
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) version 10.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
+
+You can configure SAML authentication in Grafana either through the UI or the Grafana config file.
+
+Grafana SAML UI has the following advantages over configuring SAML in the Grafana config file:
+
+- It is accessible by hosted Grafana users;
+- It doesn't require Grafana to be restarted after a configuration update;
+- Access to SAML UI only requires access to SAML settings, so it can be used by users with limited access to Grafana's configuration.
+
+Configuration in the UI takes precedence over configuration in the Grafana config file.
+
+Learn more about [SAML authentication in Grafana and how to set it up through the Grafana config file]({{< relref "../saml/" >}}).
+
```

Comment:
Nice intro. Then what we'll want to add is a Before you begin section that explains any prereqs the user should have in place before they start this task. Like so:

## Before you begin

To follow this guide, you need:

- Knowledge of SAML authentication
- Grafana permissions that allow you to xxx
- Anything else that we'd like users to have

Reply:
I've taken a stab at it, let me know what you think!

Reply:
yep, that looks really good.